src/main/java/com/iexec/core/configuration/SmsConfiguration.java

```diff
@@ -40,4 +44,9 @@
     public String getSmsURL() {
         return protocol + "://" + host + ":" + port;
     }
+
+    @Bean
```

Comment:
Compact :clap:!
As discussed we'll read/create dynamic log level in next PRs

Reply:
yes